tests/support/parser/__init__.py

```diff
@@ -241,6 +241,14 @@ def __init__(self, testsuite_directory, *args, **kwargs):
         self.output_options_group = optparse.OptionGroup(
             self, 'Output Options'
         )
+        self.output_options_group.add_option(
+            '-f',
```

Comment:
I wonder if this should be `-F`? I always think of the lowercase as `force`.

Reply:
I'm cool with that and will push this minor change.

Reply:
@cachedout Pytest uses `-X` not sure if that changes anything on your thoughts here or not. :)

https://docs.pytest.org/en/latest/usage.html#stopping-after-the-first-or-n-failures